src/js/components/Select/SelectContainer.js

```diff
@@ -86,22 +86,21 @@ const SelectContainer = forwardRef(
     ref,
   ) => {
     const theme = useContext(ThemeContext) || defaultProps.theme;
-    const [activeIndex, setActiveIndex] = useState(-1);
+    const [activeIndex, setActiveIndex] = useState(usingKeyboard ? 0 : -1);
     const [keyboardNavigation, setKeyboardNavigation] = useState();
     const searchRef = useRef();
     const optionsRef = useRef();
     const clearRef = useRef();
+    const activeRef = useRef();
 
+    // Only update active suggestion index when the mouse actually moves,
+    // not when suggestions are moving under the mouse.
+    const [mouseMovedSinceLastKey, setMouseMovedSinceLastKey] = useState(true);
```

Comment:
A couple of thoughts/questions around this variable. Will connect with you to talk through.

Reply:
I've cleaned this up now.